corehq/apps/reports/standard/monitoring.py

```diff
@@ -1428,9 +1428,11 @@ def _rows_by_user(self, report_data):
         last_form_by_user = self.es_last_submissions()
         for user in self.users_to_iterate:
             active_cases = int(report_data.active_cases_by_owner.get(user["user_id"].lower(), 0)) + \
-                sum([int(report_data.active_cases_by_owner.get(group_id, 0)) for group_id in user["group_ids"]])
+                sum([int(report_data.active_cases_by_owner.get(group_id, 0)) for group_id in user["group_ids"]]) + \
```

Comment:
Cal, sorry not quite following.  

Is the question
- Do we want to the location case in the worker activity report? [yes]
- Do we want to show cases owned by locations in the worker activity report? [also yes]

Grab me on slack if you want to discuss quickly.